home.html

```diff
@@ -0,0 +1,254 @@
+              />
+            </svg>
+          </a>
+
+          <a href="https://twitter.com/">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="16"
+              height="16"
+              fill="currentColor"
+              class="bi bi-twitter"
+              viewBox="0 0 16 16"
+            >
+              <path
+                d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"
+              />
+            </svg>
+          </a>
+
+          <a href="#">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="16"
+              height="16"
+              fill="currentColor"
+              class="bi bi-facebook"
+              viewBox="0 0 16 16"
+            >
+              <path
+                d="M16 8.049c0-4.446-3.582-8.05-8-8.05C3.58 0-.002 3.603-.002 8.05c0 4.017 2.926 7.347 6.75 7.951v-5.625h-2.03V8.05H6.75V6.275c0-2.017 1.195-3.131 3.022-3.131.876 0 1.791.157 1.791.157v1.98h-1.009c-.993 0-1.303.621-1.303 1.258v1.51h2.218l-.354 2.326H9.25V16c3.824-.604 6.75-3.934 6.75-7.951z"
+              />
+            </svg>
+          </a>
+          <span>Logout</span>
+          <span>MyPage</span>
+          <span>English</span>
+        </div>
+      </header>
+
+      <div class="my-logo">
+        <a href="home.html"
+          ><img class="logo-img" src="images/mylogo(2).PNG" alt="resto_logo"
+        /></a>
+        <a href="#"><p class="resto-name">Chef Food Restaurant</p></a>
+      </div>
+      <div class="container" id="hamburger">
+        <div class="bar1"></div>
+        <div class="bar2"></div>
+        <div class="bar3"></div>
+      </div>
+      <ul id="nav-links">
+        <li>
+          <a href="home.html">Home</a>
+        </li>
+        <li class="text">
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="#">Program</a>
+        </li>
+        <li>
+          <a href="#">Join</a>
+        </li>
+        <li class="border-line">
+          <a href="#"> Day Menu</a>
+        </li>
+      </ul>
+    </nav>
+    <section class="intro">
+      <div class="intro-content">
+        <h2 class="intro-message">"Your Satisfaction Our Pride"</h2>
+        <h1>Welcome to Chef Food Restaurant</h1>
+      </div>
+      <div class="intro-desc">
+        <p>
```

Comment:
- [ ] In the mobile screen under nearly 450-width view, the design breaks. Please fix it to match the given design.
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/169448097-5db6eac7-1c4e-44b2-ad1b-a85bfd80ed52.png)

Reply:
Hello @MahmoudBakr23 
    Just looked back at my design and i think it's ok. But will be helpful if you elaborate more on your point of my mobile design breaking under nearly 450-width view. Just checked it and it was very ok. It starts breaking at 350 which i think is quite a very small width. The image couldn't be responsive to that level. As it has it minimum screen.

Reply:
Hello @MahmoudBakr23